#!/bin/sh /usr/share/dpatch/dpatch-run
## 205-magic-update-tcsh.dpatch
##
## DP: Add another shebang path for tcsh.

@DPATCH@

diff -Naurp file.orig/magic/Magdir/commands file/magic/Magdir/commands
--- file.orig/magic/Magdir/commands	2008-06-13 09:30:59.000000000 +0200
+++ file/magic/Magdir/commands	2008-06-13 09:31:50.000000000 +0200
@@ -12,6 +12,8 @@
 !:mime	application/x-shellscript
 0	string/b 	#!\ /bin/tcsh		Tenex C shell script text executable
 !:mime	application/x-shellscript
+0	string/b	#!\ /usr/bin/tcsh	Tenex C shell script text executable
+!:mime application/x-shellscript
 0	string/b 	#!\ /usr/local/tcsh	Tenex C shell script text executable
 !:mime	application/x-shellscript
 0	string/b	#!\ /usr/local/bin/tcsh	Tenex C shell script text executable
